### PR TITLE
Disable OrderColumns in VariationMart_conf

### PR DESCRIPTION
**Pull request by radome**
Kind of testing if mart building will be faster overall for 103, without ordering by column

### CODE REVIEW

**Review by marcoooo — COMMENTED** [review]
I would have complicated a little the pipeline, and done this skip only with human species. considering the others don't seem to be too much impacted.

**Comment by radome** [comment]
> I would have complicated a little the pipeline, and done this skip only with human species. considering the others don't seem to be too much impacted.

Why adding complexity if the expected gain is supposed to be minimal? Also, this patch should actually serve as a test to measure overall runtime without the OrderColumns step.

File: modules/Bio/EnsEMBL/EGPipeline/PipeConfig/VariationMart_conf.pm [hunk 1]
```diff
@@ -430,18 +430,18 @@ sub pipeline_analyses {
                             },
       -max_retry_count   => 0,
       -analysis_capacity => 10,
-      -flow_into         => ['CreateDependentTables'],
```
Comment (james-monkeyshines) [inline modules/Bio/EnsEMBL/EGPipeline/PipeConfig/VariationMart_conf.pm:433]: I think this was changed to 'CreateMartIndexes' by mistake?

**Review by james-monkeyshines — COMMENTED** [review]
I think it's fine to remove the analysis completely and see how the pipeline performs - the idea behind doing the ordering in the first place was to speed up the indexing. If it turns out that the indexing for other species has slowed down a lot, can always add it in selectively.

File: modules/Bio/EnsEMBL/EGPipeline/PipeConfig/VariationMart_conf.pm (hunk 1 above)
Comment (radome) [reply modules/Bio/EnsEMBL/EGPipeline/PipeConfig/VariationMart_conf.pm:433]: Yep, my bad. Thanks, well spotted!

**Review by james-monkeyshines — APPROVED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**